Lib/test/test_unicode.py

```diff
@@ -861,12 +866,13 @@ def test_swapcase(self):
 
     def test_center(self):
         string_tests.CommonTest.test_center(self)
-        self.assertEqual('x'.center(2, '\U0010FFFF'),
-                         'x\U0010FFFF')
-        self.assertEqual('x'.center(3, '\U0010FFFF'),
-                         '\U0010FFFFx\U0010FFFF')
-        self.assertEqual('x'.center(4, '\U0010FFFF'),
-                         '\U0010FFFFx\U0010FFFF\U0010FFFF')
+        ch = LAST_CHAR
```

Comment:
Seems it is just an arbitrary non-ASCII character.

Ideally, this test should be repeated for different string kinds: ASCII, UCS1, UCS2, UCS4, and for different combinations of arguments (totally `4*4 = 16` combinations).